crates/ironrdp-web/src/session.rs

```diff
@@ -818,6 +829,33 @@ impl Session {
         // plain scancode events are allowed to function correctly).
         false
     }
+
+    pub fn extension_call(&self, value: JsValue) -> Result<JsValue, RemoteDesktopError> {
+        match serde_wasm_bindgen::from_value::<SessionExtensionCall>(value)
+            .map_err(|err| anyhow::anyhow!("provided JsValue is not a valid extension call: {err:?}"))?
```

Comment:
style: Can be simplified:

```suggestion
        match serde_wasm_bindgen::from_value::<SessionExtensionCall>(value)
            .context("provided JsValue is not a valid extension call")?
```

Reply:
Done